pyproject.toml

```
@@ -11,9 +11,9 @@ python = "^3.12"
 Celery = "==5.2.7"
 crispy-forms-gds = "^0.2.3"
 dataclasses-json = "==0.5.7"
-django = "^4.1.10"
+django = "^4.2"
 djangorestframework = "^3.12.4"
-django-hawk-drf = "1.1.0"
+django-hawk-drf = "1.1.2"
```

Can we update this to 
```suggestion
django-hawk-drf = "^1.1.2"
```